File: JAVA/Car.java
package JAVA;

public class Car {
    Integer id;
    String license;
    Account driver;
   private Integer passegenger;

public Car(String license,Account driver){
    this.license=license;
    this.driver = driver;

}
    void printDataCar() {
        System.out.println("License: " + license + " Driver: " + driver.name);
    }
    Integer getPassegenger(){
        return passegenger;
    }
    void setPassegenger(Integer passegenger){
this.passegenger = passegenger;
    }
}

File: PHP/Car.php
<?php
class Car {
    public $id;
    public $license;
    public $driver;
    public $passengers;
  
    public function __construct($license, $driver) {
      $this->license = $license;
      $this->driver = $driver;
    }
  
    public function PrintDataCar(){
      echo "La licencia es: $this->license, conductor: {$this->driver->name}, documento: {$this->driver->document}";
    }
  }